docs/frontend/stimulus.md

```diff
@@ -0,0 +1,147 @@
+# Stimulus
+
+For out javascript needs we use Symfony UX with Stimulus. See the Symfony site for information https://symfony.com/bundles/StimulusBundle/current/index.html or the stimulus documentation for more information: https://stimulus.hotwired.dev/handbook/introduction
+
+We have a few default components that can be used in your project:
+
+## Clipboard
+
+Je kan deze op twee manieren gebruiken:
```

Comment:
Alles in EN zetten? Of wat is de default voor de docs?

Reply:
Fixed ecca52c